config.yaml

```diff
@@ -16,8 +16,8 @@ streams:
   #  type: mechanical
   #  env:
   #    COSA_USE_OSBUILD: true
-  # bodhi-updates:
-  #   type: mechanical
+  bodhi-updates:
+    type: mechanical
```

Comment:
hmm. is this a requirement? I'm not actually sure new builds on this stream would succeed.

Reply:
afaik, we would need this to have it as a valid stream option when running the [GC](https://jenkins-fedora-coreos-pipeline.apps.ocp.fedoraproject.org/blue/organizations/jenkins/garbage-collection/activity/).

Reply:
If everything runs okay, everything should be pruned off in the initial run for this stream and we can disable it again.

Reply:
instead of doing this let's just [modify the job](https://github.com/coreos/fedora-coreos-pipeline/blob/b60103ffb8b66f45032ae9712df081b49507715c/jobs/garbage-collection.Jenkinsfile#L17) to hardcode `bodhi-updates` in the list of available streams to operate on for now.

I really don't want to set this because I don't want the pipeline daily trying to build bodhi updates.